schematic/models/validate_attribute.py

```diff
@@ -1262,15 +1325,18 @@ def _parse_validation_log(
         return invalid_rows, invalid_entries, manifest_ids
 
     def _merge_format_invalid_rows_values(
-        self, series_1: pd.core.series.Series, series_2: pd.core.series.Series
-    ) -> tuple[[list[str], list[str]]]:
+        self, series_1: pd.Series, series_2: pd.Series
```

Comment:
nit: The name of the arguments might be a bit clearer with a name update to match the argument description. Like `series_1` -> `invalid_values_1` or something like that.

Reply:
I'm going to let someone else clean this method up in the future.